network-api/networkapi/templates/pages/buyersguide/about/press.html

```diff
@@ -10,200 +10,222 @@ <h1 class="tw-h2-heading">
+                guide’s groundbreaking product reviews scrutinize the privacy features and flaws of <b>connected gifts, sex toys, dating
+                    apps, reproductive health apps, mental health apps,</b> and <b>cars</b>.
+            {% endblocktrans %}
         </p>
+        <div class="tw-my-14">
+            <iframe class="tw-mx-auto" width="560" height="315" src="https://www.youtube.com/embed/XKQ-uxTw11g?si=XuG0CsQdaM3_dsxm" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+        </div>
         <p class="tw-body">
-            {% blocktrans with url="https://www.dailydot.com/irl/mozilla-privacy-not-included-sex-toys-dating-apps/" trimmed %}
-                <strong>Daily Dot:</strong> <a href="{{ url }}">These sex toys and dating apps may jeopardize your privacy, Mozilla warns </a>
+            {% blocktrans with url="https://www.topgear.com/car-news/tech/modern-cars-branded-privacy-nightmare-mozilla" trimmed %}
+                <strong>BBC Top Gear:</strong> <a href="{{ url }}">Modern cars branded a ‘privacy nightmare’ by Mozilla</a>
             {% endblocktrans %}
             <br>
             {% blocktrans trimmed %}
-                Published February 2021. Writes The Daily Dot: “Mozilla’s new Privacy Not Included report looks at 50 sex toys and
-                dating apps and ranks them based on how they respect users’ privacy. And the answers aren’t sexy.”
-
+                Published September 2023. “First up, Mozilla says every car brand collects “too much personal data”... Secondly, the
+                survey concluded that 84 per cent of motor manufacturers share your data and a worrying 76 per cent sell drivers’
+                personal data.”
             {% endblocktrans %}
         </p>
+        <div class="feature-quote my-4 py-2">
+            <h3>
+                {% blocktrans trimmed %}
+                    For generations, the car in America has come to symbolize freedom… And for many, freedom and privacy go hand in hand.
+                    It's hard to have one without the other. But according to a new report, your car could be worse than your phone when it
+                    comes to privacy.
+                {% endblocktrans %}
+            </h3>
+            <div class="attribution-with-link tw-inline-flex tw-gap-2">
+                {% blocktrans with url="https://www.npr.org/2023/09/16/1200012617/experts-say-modern-cars-can-violate-consumer-privacy" trimmed %}
+                    <p class="tw-h6-heading ">
+                        — Scott Detrow,
+                    </p>
+                    <a class="tw-blue-80 tw-text-xs" href="{{ url }}">
+                        NPR
+                    </a>
+                {% endblocktrans %}
```

Comment:
This one doesn't seem like it needs a `blocktrans` unless we need to localise "NPR". In that case, I would put just a translation tag around NPR instead of a `blocktrans` for the whole block

Reply:
Great catch @jhonatan-lopes! I have updated this in my latest commit.